iroh/src/node/builder.rs

```diff
@@ -85,7 +94,7 @@ where
     gc_policy: GcPolicy,
     dns_resolver: Option<DnsResolver>,
     node_discovery: DiscoveryConfig,
-    docs_store: iroh_docs::store::Store,
+    docs_store: Option<DocsStorage>,
```

Comment:
This just says where the docs are stored, it is no longer a store.

Reply:
changed!